pandas/io/formats/css.py

```diff
@@ -172,7 +187,9 @@ def __call__(self, declarations_str, inherited=None):
 
     def size_to_pt(self, in_val, em_pt=None, conversions=UNIT_RATIOS):
         def _error():
-            warnings.warn(f"Unhandled size: {repr(in_val)}", CSSWarning)
+            warnings.warn(
+                f"Unhandled size: {repr(in_val)}", CSSWarning,
```

Comment:
```suggestion
                f"Unhandled size: {in_val}", CSSWarning,
```

Shouldn't need the `repr` here any more

Reply:
I see marked as resolved but I think this is open; if you can commit changes or comment as to why not needed would be helpful